Otter farming solutions.ipynb

```diff
@@ -190,19 +190,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 5,
```

Comment:
Shouldn't push changes to this file, it is supposed to serve a as a local testing environment